vagrant/env.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-export PATH="/home/vagrant/python/bin:$PATH"
+export PATH="$HOME/firefox:$HOME/python/bin:$PATH"
```

Comment:
I know this is pre-existing, but I don't suppose you could convert this to using the brace style? (I'm slowly trying to transition towards it, since it's generally considered best practice - eg https://google.github.io/styleguide/shell.xml#Variable_expansion)

So:
`export PATH="${HOME}/firefox:${HOME}/python/bin:${PATH}"`